source/extensions/filters/http/cache/cache_filter.cc

```diff
@@ -37,13 +37,17 @@ Http::FilterHeadersStatus CacheFilter::decodeHeaders(Http::RequestHeaderMap& hea
         *decoder_callbacks_, headers);
     return Http::FilterHeadersStatus::Continue;
   }
-  if (!CacheFilterUtils::isCacheableRequest(headers)) {
+  if (!CacheabilityUtils::isCacheableRequest(headers)) {
     ENVOY_STREAM_LOG(debug, "CacheFilter::decodeHeaders ignoring uncacheable request: {}",
                      *decoder_callbacks_, headers);
     return Http::FilterHeadersStatus::Continue;
   }
   ASSERT(decoder_callbacks_);
-  lookup_ = cache_.makeLookupContext(LookupRequest(headers, time_source_.systemTime()));
+
+  auto lookup_request = LookupRequest(headers, time_source_.systemTime());
```

Comment:
No need for auto here:
LookupRequest lookup_request(headers, time_source_.systemTime());

Reply:
Edited.